File: iniciante/python/1101.py
# -*- coding: utf-8 -*-

M = 1
N = 1
while True:
    M, N = input().split(" ")
    M = int(M)
    N = int(N)
    if M <= 0 or N <= 0:
        break
    Sum = 0
    elementos =""
    if M < N:
        for i in range(M, N+1,1):
            Sum += i
            if i !=  M:
                elementos = elementos + " " + str(i)
            else:
                elementos = elementos + str(i)
        print(elementos, end=" Sum=%d" %Sum)
        print()
    else:
        for i in range(N, M+1,1):
            Sum += i
            if i != N:
                elementos = elementos + " " + str(i)
            else:
                elementos = elementos + str(i)
        print(elementos, end=" Sum=%d" %Sum)
        print()

File: iniciante/c/1101.c
#include <stdio.h>
 
int main() {

	int M, N, i, Sum;

	M = 1;
	N = 1;
	while (M > 0 && N > 0){
		scanf("%d", &M);
		scanf("%d", &N);
		Sum = 0;
		if (M > N && (M > 0 && N > 0)){
			for (i =N; i <= M; i++){
				printf("%d ", i);
				Sum += i;
				if (i == M){
					printf("Sum=%d\n", Sum);
				}
			}
		}
		if (N > M  && (M > 0 && N > 0)){
			for (i=M; i<= N; i++){
				printf("%d ", i);
				Sum += i;
				if (i == N && (M > 0 && N > 0)){
					printf("Sum=%d\n", Sum);
				}
			}
		}
	}


    return 0;
}